Monika After Story/game/script-ch30.rpy

```diff
@@ -741,7 +724,7 @@ init 1 python:
 #       NOTE: this must be a string
 #       NOTE: if passed in, this will override aff-based exps from dissolving.
 #       (Default: None)
-label spaceroom(start_bg=None, hide_mask=False, hide_monika=False, dissolve_all=False, dissolve_masks=False, scene_change=False, force_exp=None):
+label spaceroom(start_bg=None, hide_mask=store.mas_current_background.hide_masks, hide_monika=False, dissolve_all=False, dissolve_masks=False, scene_change=False, force_exp=None, hide_calendar=store.mas_current_background.hide_calendar, day_bg=store.mas_current_background.getDayRoom(), night_bg=store.mas_current_background.getNightRoom()):
```

Comment:
python named args like these are resolved on init and don't change dynamically, so if the current bg changes these params wont update. Even if renpy actualyl does this dynamically, should just use None for dynamic cases so no one looks at this and tries to do the same in py.

So have these default to None, then after the `with None`, check those args for None values and set to defaults accordingly.